pandas/core/dtypes/base.py

```diff
@@ -109,6 +109,11 @@ class ExtensionDtype(_DtypeOpsMixin):
     * name
     * construct_from_string
 
+    Optionally one can override construct_array_type for construction
+    with the name of this dtype via the Registry
```

Comment:
I would give some additional explanation of what the Registry is, because now this is not explained here?